frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -619,6 +619,14 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
   props,
 ) => {
   const { spec, metadata, status } = props.obj;
+  const ibmSupportLink = _.get(props.obj, [
```

Comment:
This path is wrong if you have the ClusterServiceVersion resource. This path is for the PackageManfiest.

Here it would just be on `metadata.annotations`.